site/docs/main/troubleshooting/operator.md

```diff
@@ -0,0 +1,91 @@
+# Operator Troubleshooting
```

Comment:
it would be nice if we could keep operator docs in the operator repo itself, and then somehow pull them into the main projectcontour.io site, rather than having to put the operator docs in the main contour repo. I'm not sure if we have a way to do this in our current Jekyll setup, or our possible future Hugo setup, but maybe @jonasrosland has an idea. Don't need to tackle in this PR, just putting the thought down.

Reply:
👍

Reply:
We talked initially at one point of not having the site inside Contour repo at all. This might be a good alternative as well to just move the site to it's own spot to start managing all the various bits.